src/content/content.controller.ts

```diff
@@ -193,4 +195,54 @@ export class ContentController {
       tagName: selectedArticleSlugsByTag.tagName,
     }
   }
+
+  @Get('blog-articles-by-slug/:slug')
+  async findBlogArticlesBySlug(@Param('slug') slug: string) {
+    const blogArticles: BlogArticleAugmented[] =
+      await this.contentService.findByType(
+        ContentType.blogArticle,
+        undefined,
+        undefined,
+        {
+          data: {
+            path: ['slug'],
+            equals: slug,
+          },
+        },
+      )
```

Comment:
Can we change the signature of this `findByType` to accept an "options" object instead of an ordered list of args so that we can avoid using `undefined` directly in this way? If we find ourselves w/ a method that's accepting more than 2 or 3 args, that's typically a code-smell that either the method is trying to do too much, or that taking an options object would be a more flexible pattern

```suggestion
      await this.contentService.findByType({
        type: ContentType.blogArticle,
        where: {
          data: {
            path: ['slug'],
            equals: slug,
          },
        }
      })
```